### DISCUSSION TITLE
Testing when audio system was crashed

### DISCUSSION THREAD

**Discussion by BKVad1m** [ppy/osu#20615] [Ideas]
https://user-images.githubusercontent.com/76803825/194704629-1f69074b-0afe-42c9-b929-33d15b18b9c1.mp4

make this feature notification alert

**Comment by peppy** [comment]
I'm sorry i have no idea what this is showing. Could you please include logs?

**Reply by frenzibyte** [reply]
Logs is still required but it seems the audio device in OP has stopped working and the track playback froze.

**Reply by nagi-desuuu** [reply]
I vouch for the OP because this happens to me all the time. Although it's kinda obvious that I have it running under a system with sub-par performance, so it was inevitable that it would happen eventually.

**Comment by BKVad1m** [comment]
![image](https://user-images.githubusercontent.com/76803825/194711778-9b9e55c6-ea1b-499a-a619-16a98bab3681.png)
this log running only startup

**Reply by peppy** [reply]
Please provide the *full log files* as requested.
